bless/aws_lambda/bless_lambda_lyft_host.py

```diff
@@ -0,0 +1,303 @@
+"""
+.. module: bless.aws_lambda.bless_lambda
```

Comment:
This is copied almost entirely from the old forked version that was being used for hosts (about 4 years old).  Its been modernized for Python3 and a few tweaks caused by changes to interfaces in the bless package, but not refactored otherwise